sources/packages/backend/apps/db-migrations/src/sql/CASSuppliers/Add-errors-column.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE
+    sims.cas_suppliers
+ADD
+    COLUMN errors VARCHAR(300) [];
+
+COMMENT ON COLUMN sims.cas_suppliers.errors IS 'Errors received from CAS.';
```

Comment:
Please adjust to "Errors while execution CAS verifications.".
The errors will also be generated by SIMS even without calling CAS.